src/Classes/Particle.ts

```diff
@@ -219,10 +225,22 @@ export class Particle {
     }
 
     private calcPosition(container: Container, position?: ICoordinates): ICoordinates {
-        const pos = {
-            x: position && position.x ? position.x : Math.random() * container.canvas.width,
-            y: position && position.y ? position.y : Math.random() * container.canvas.height,
-        };
+        let pos = {x: 0, y: 0};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'pos' is never reassigned; use 'const' instead of 'let'.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5042590)